io_chan.go

```diff
@@ -1,20 +1,29 @@
 package main
 
-import "io"
+import (
+	"io"
+	"time"
+)
+
+// Simulates a TCP packet by storing a slice of bytes with the receive timestmap
```

Comment:
This is actually part of https://github.com/Shopify/toxiproxy/pull/47, just the second commit is for this PR.